### ISSUE TITLE
Scale icons with accessibility text size on iOS

### ISSUE THREAD

**Issue by devinjameson** [react-native-svg/react-native-svg#1549]
# Feature Request

Is it possible to scale icons with the accessibility text size set in iOS settings? This is considered best practice so text can't be huge relative to icons, as the two often go together.

## Why it is needed

See screenshot below. This is a screenshot of an app with accessibility text size set to the largest possible size. Icons should be able to scale with accessibility font size.

<img width="305" alt="Screen Shot 2021-02-27 at 6 05 44 PM" src="https://user-images.githubusercontent.com/39350030/109402752-801c7980-7926-11eb-8797-596bcc1e1087.png">

## Possible implementation

Is there a way to hook into the accessibility text size set in Settings and scale icons accordingly if `SvgXml` received a certain prop e.g. `scaleWithAccessibilityTextSize`.

**Comment by SaruwatariSagittario** [comment]
any updates regarding these, also in need to scale the vector to keep up with the text
